tests/RME/gold_hrrr/config.ini

```diff
@@ -307,8 +308,8 @@ input_backup:                  False
 ################################################################################
 
 [logging]
-log_file:                      /home/micahsandusky/Code/smrf/tests/RME/output/log.txt
 log_level:                     debug     
+log_file:                      None      
```

Comment:
We should add the file name here so it doesn't print out during testing.

Reply:
This is the gold standard backup file so it shouldn't be ran by anyone, it's more here to be reproducible.

Reply:
Ah gotcha. I didn't pay attention to the path and I assumed it was for running the test.